velox/dwio/common/Options.h

```diff
@@ -609,6 +609,8 @@ struct WriterOptions {
 
   std::optional<velox::common::CompressionKind> compressionKind;
   std::optional<uint64_t> orcMinCompressionSize{std::nullopt};
+  std::function<std::unique_ptr<dwio::common::FlushPolicy>()>
+      flushPolicyFactory;
```

Comment:
Is it a required parameter when writing files and what happens if it is not correctly set by the user?

Reply:
@rui-mo No. It is not required. If not set by user, it will use the DefaultFlushPolicy.
https://github.com/facebookincubator/velox/blob/main/velox/dwio/parquet/writer/Writer.cpp#L234
https://github.com/facebookincubator/velox/blob/main/velox/dwio/dwrf/writer/Writer.cpp#L87